pkg/model/keycloak_deployment.go

```diff
@@ -208,11 +208,13 @@ func KeycloakDeployment(cr *v1alpha1.Keycloak, dbSecret *v1.Secret, dbSSLSecret
 		"component": KeycloakDeploymentComponent,
 	}
 	podLabels := AddPodLabels(cr, labels)
+	podAnnotations := AddPodAnnotations(cr, cr.Annotations)
```

Comment:
I think we should not be copying annotations from the CR's annotation. Should be something like this:
```suggestion
	podAnnotations := cr.Spec.KeycloakDeploymentSpec.PodAnnotations
```

Reply:
thanks for hint, I just thought treating it like in the labels

Reply:
changed it also for `rhsso_deployment`